src/courtroom.cpp

```diff
@@ -4265,10 +4265,21 @@ void Courtroom::chat_tick()
     ui_vp_chat_arrow->setTransformationMode(ao_app->get_misc_scaling(f_custom_theme));
     ui_vp_chat_arrow->loadAndPlayAnimation("chat_arrow", f_custom_theme); // Chat stopped being processed, indicate that.
     QString f_message_filtered = filter_ic_text(f_message, true, -1, m_chatmessage[TEXT_COLOR].toInt());
-    for (int c = 0; c < max_colors; ++c)
-    {
-      additive_previous = additive_previous.replace("$c" + QString::number(c), char_color_rgb_list.at(c).name(QColor::HexRgb));
-      f_message_filtered = f_message_filtered.replace("$c" + QString::number(c), char_color_rgb_list.at(c).name(QColor::HexRgb));
+    if (Options::getInstance().customChatboxEnabled())
+    { // chatbox colors
+      for (int c = 0; c < max_colors; ++c)
+      {
+        additive_previous = additive_previous.replace("$c" + QString::number(c), char_color_rgb_list.at(c).name(QColor::HexRgb));
```

Comment:
Isn't this and the one below literally the same function call or am I missing something here?

Reply:
If custom chat boxes are enabled, it pulls data from char_color_rgb_list, otherwise it's pulled from default_color_rgb_list